worldedit-forge/build.gradle

```diff
@@ -57,7 +57,7 @@ minecraft {
         }
     }
 
-    accessTransformer = file('worldedit_at.cfg')
+    //accessTransformer = file('worldedit_at.cfg')
```

Comment:
Can we just drop the AT configuration for now? If we end up needing in in 1.15, etc. we can add it back. It doesn't seem worth of keeping it tracked while commented.